### PR TITLE
created dash

### PR DESCRIPTION
**Pull request by msp2248**

### CODE REVIEW

File: src/pages/DriverDash.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+              <div className="img" style={{'margin-bottom':'50px'}}>
+                  <img class="card-img-top" src={require('../media/Larry.png')} alt="Card image"></img>
+              </div>
+              <div className = "text_box" style={{display:'inline-block','vertical-align':'left'}}>
+                      <p className="card-text" align="top" style={{'margin-top':'10px'}}><strong>Age: </strong>{json['user1'].age}</p>
+                      <p className="card-text" align="top"><strong>License Number: </strong>{json['user1'].license}</p>
+                      <p className="card-text" align="top"><strong>Plate Number: </strong>{json['user1'].plate}</p>
+                      <p className="card-text" align="top"><strong>Insurance Provider: </strong>{json['user1'].insurance}</p>
+              </div>
+            </div>
```
Comment (cxs7700) [inline src/pages/DriverDash.js]: Just wanted to note that the driver information doesn't match the Figma wireframes but that's fine. You could have also used the Larry component for this.

File: src/pages/DriverDash.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+              <p className="card-text"><strong>Time: </strong>{json['trip2'].time}</p>
+              <p className="card-text"><strong>Departure: </strong>{json['trip2'].departure}</p>
+              <p className="card-text"><strong>Destination: </strong>{json['trip2'].destination}</p>
+              <p className="card-text"><strong>Spots Open: </strong>{json['trip2'].spotsOpen}</p>
+              <div class="row">
+                <div class="col text-center">
+                  <a href="/trip" className="btn btn-dark">
+                    View Trip Details
+                  </a>
+                </div>
```
Comment (cxs7700) [inline src/pages/DriverDash.js]: Should be "View Ride Requests" and href should be "/ride-requests-1" to render the ViewRequests component, not the Trip component.

**Review by cxs7700 — CHANGES_REQUESTED** [review]

File: src/pages/DriverDash.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+              <p className="card-text"><strong>Date: </strong>{json['trip2'].date}</p>
+              <p className="card-text"><strong>Time: </strong>{json['trip2'].time}</p>
+              <p className="card-text"><strong>Departure: </strong>{json['trip2'].departure}</p>
+              <p className="card-text"><strong>Destination: </strong>{json['trip2'].destination}</p>
+              <p className="card-text"><strong>Spots Open: </strong>{json['trip2'].spotsOpen}</p>
+              <div class="row">
+                <div class="col text-center">
+                  <a href="/trip" className="btn btn-dark">
+                    View Trip Details
+                  </a>
```
Comment (jeremydelacruz) [inline src/pages/DriverDash.js]: Same as previous comment; try using `<Link>`

File: src/pages/DriverDash.js [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+              <p className="card-text"><strong>Date: </strong>{json['trip1'].date}</p>
+              <p className="card-text"><strong>Time: </strong>{json['trip1'].time}</p>
+              <p className="card-text"><strong>Departure: </strong>{json['trip1'].departure}</p>
+              <p className="card-text"><strong>Destination: </strong>{json['trip1'].destination}</p>
+              <p className="card-text"><strong>Spots Open: </strong>{json['trip1'].spotsOpen}</p>
+              <div class="row">
+                <div class="col text-center">
+                  <a href="/trip" className="btn btn-dark">
+                    View Trip Details
+                  </a>
```
Comment (jeremydelacruz) [inline src/pages/DriverDash.js]: Don't think using a normal `href="/trip"` would work with our react-router implementation. Try using something like: 
`<Link to="/driverdash">View Trip Details</Link>`
See other examples in other pages

File: src/pages/DriverDash.js [hunk 5]
```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+
+class DriverDash extends React.Component {
```
Comment (jeremydelacruz) [inline src/pages/DriverDash.js:3]: Should we rename DriverDash to something else? Maybe just dashboard? This doesn't need to be addressed with this PR but something to think about bc user tests revealed that people were kinda confused on what it meant

**Review by jeremydelacruz — CHANGES_REQUESTED** [review]

**Review by jeremydelacruz — APPROVED** [review]

File: src/pages/DriverDash.js (hunk 5 above)
Comment (cxs7700) [reply src/pages/DriverDash.js:3]: We can just rename it to that yeah. The button could be renamed to Dashboard, but we would need to disable the button for Riders (i.e. separate home page using your PrivateRoute).

File: src/pages/DriverDash.js (hunk 3 above)
Comment (msp2248) [reply src/pages/DriverDash.js]: Oh ok got it

File: src/pages/DriverDash.js (hunk 1 above)
Comment (msp2248) [reply src/pages/DriverDash.js]: Ok got it

**Pull request merged**